teamwork/index.md

```diff
@@ -0,0 +1,29 @@
+# Teamwork (makes the dream work)
+
+Programming is a team sport. We adhere to the following practices to make working together as enjoyable as possible.
+
+## Small prs
+
+Try to keep PRs as small as possible. There are no hard and fast rules here, but anything over 1,000 lines can probably be split up into smaller parts.
+
+You can think of a pr as a unit of self-contained work. So if a PR is getting long, ask yourself if it could be split up into smaller units of self-contained work.
+
+If you do **have** to submit a large pr (you probably don't), leave a note at the top explaining why. (Especially true if there are a large number of automatically generated diffs in your pr, e.g. adding a new package or lamda: this gives reviewers a better sense of the "true" size of the pr).
+
+## Commits and commit messages
+
+Keep commits on-topic. Ideally, a commit does one, specific thing, and that thing is described accurately by the commit message.
```

Comment:
Could we be a bit more specific about what the good and bad commit message look like, e.g. 

Bad: `Refactor: Cleanup`
Good: `Refactor: Cleanup imports in ColonyTokenSelector`

and so on...

A good commit message indicates what change occurred where, so that it's easy to locate if there's a need to investigate/revert it. Not that you haven't already said that, but sometimes you need to be very explicit, especially with new hires 🙂

Reply:
I was looking through past CDapp's commit and @ArmandoGraterol's ones really stand out:
<img width="510" alt="image" src="https://user-images.githubusercontent.com/112586815/230180245-1be6f8e5-6f0b-452c-a3f3-b458555deea5.png">
Atomic, clear and precise!